craft_parts/overlays/chroot.py

```diff
@@ -150,4 +153,7 @@ def _cleanup_chroot_linux(path: Path) -> None:
 
         if mountpoint.exists():
             logger.debug("[pid=%d] umount: %r", pid, str(mountpoint))
-            os_utils.umount(str(mountpoint))
+            if entry.option == "--rbind":
+                os_utils.umount(str(mountpoint), "--recursive", "--lazy")
```

Comment:
why is lazy required?

Reply:
It covers the case where a filesystem under /dev is busy (e.g. /dev/shm being used), in which case /dev becomes unmountable. This shouldn't be the case under normal conditions when using a build instance, however it may happen in destructive mode in certain environments such as the github test machines. In practice it sets the `MNT_DETACH` flag in umount(2) to defer actual unmounting, and should be safe to use in our scenario.

Reply:
Adding a comment to the source code.